backend/src/api/fundraiser/fundraiser.services.ts

```diff
@@ -108,3 +113,64 @@ export const createFundraiser = async (
 
   return fundraiser;
 };
+
+export const updateFundraiser = async (
+  fundraiserBody: UpdateFundraiserBody & { fundraiserId: string }
+) => {
+  const fundraiser = await prisma.fundraiser.update({
+    where: {
+      id: fundraiserBody.fundraiserId,
+    },
+    data: {
+      name: fundraiserBody.name,
+      description: fundraiserBody.description,
+      imageUrls: fundraiserBody.imageUrls,
+      startsAt: fundraiserBody.startsAt,
+      endsAt: fundraiserBody.endsAt,
+    },
+    include: {
+      organization: true,
+    },
+  });
+
+  return fundraiser;
+};
+
+export const createFundraiserItem = async (
+  itemBody: CreateFundraiserItemBody & { fundraiserId: string }
+) => {
+  const item = await prisma.item.create({
+    data: {
+      name: itemBody.name,
+      description: itemBody.description,
+      price: itemBody.price,
+      imageUrl: itemBody.imageUrl,
+      fundraiser: {
+        connect: {
+          id: itemBody.fundraiserId,
+        },
+      },
+    },
+  });
+
+  return item;
+};
+
+export const updateFundraiserItem = async (
+  itemBody: UpdateFundraiserItemBody & { itemId: string; fundraiserId: string }
+) => {
+  const item = await prisma.item.update({
+    where: {
+      id: itemBody.itemId,
+      fundraiserId: itemBody.fundraiserId,
+    },
+    data: {
+      name: itemBody.name,
+      description: itemBody.description,
+      price: itemBody.price,
+      imageUrl: itemBody.imageUrl,
+    },
```

Comment:
Make sure you update this function in accordance with the other changes to UpdateFundraiserItemBody